### DISCUSSION TITLE
Combination lookup fail in hop server

### DISCUSSION THREAD

**Discussion by RaulRoPer** [apache/hop#4504] [General]
In hop 2.10 the bug is not fixed.
When building the zip that sends to the server it fails with those dimension/combination looup components.
https://github.com/apache/hop/issues/3699

**Comment by hansva** [comment]
Not sure why this is worth a discussion. As you correctly linked there is a ticket that is not closed...

**Comment by bamaer** [comment]
+1, let's not use discussions to duplicate tickets. if you need an urgent fix for an open ticket, consider contacting a developer/SI to get a support contract.
